### PR TITLE
Add kaniko-executer image

### PR DESCRIPTION
**Pull request by moelsayed**
**Description**

Changes proposed in this pull request:

- ...
- ...
- ...

**Related issue(s)**

### CODE REVIEW

**Review by pPrecel — DISMISSED** [review]
/meow LGTM

File: kaniko-executer/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# kaniko-executer 
+
+This folder contains the kaniko-executer image that is based on the official [here](https://github.com/GoogleContainerTools/kaniko/blob/v1.6.0/deploy/Dockerfile).
```
Comment (NHingerl) [inline kaniko-executer/README.md:3]:
```suggestion
This folder contains the custom kaniko-executer image that is based on the official [image](https://github.com/GoogleContainerTools/kaniko/blob/v1.6.0/deploy/Dockerfile).
```

File: kaniko-executer/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+FROM golang:1.16-bullseye
+
+ARG KANIKO_VERSION
+
+ARG GOARCH=amd64
+WORKDIR /go/src/github.com/GoogleContainerTools/kaniko
+
+RUN echo $GOARCH > /goarch
+
+#This arg is passed by docker buildx & contains the platform info in the form linux/amd64, linux/ppc64le etc.
```
Comment (NHingerl) [inline kaniko-executer/Dockerfile:27]:
```suggestion
#This arg is passed by docker buildx and contains the platform info in the form linux/amd64, linux/ppc64le etc.
```

**Review by NHingerl — CHANGES_REQUESTED** [review]

**Review by a-thaler — APPROVED** [review]

**Review by NHingerl — APPROVED** [review]

**Pull request merged**